cpp/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 option(USE_NVTX "Build with NVTX support" ON)
 option(BUILD_TESTS "Configure CMake to build tests" ON)
 option(BUILD_BENCHMARKS "Configure CMake to build (google) benchmarks" OFF)
+option(BUILD_NVBENCH "Configure CMake to build NVBench" OFF)
```

Comment:
Is the idea that rmm benchmarks will switch over to `nv_bench`? If so we shouldn't have a CMake separate argument, and instead fetch `nv_bench` based on the value of `BUILD_BENCHMARKS`

Reply:
Yes, I think the idea is to finally replace Google Benchmark with NVBench. I don't have a good guess about how long this switching process will take so the idea is to keep both dependencies until we completely get rid of Google Benchmark. @jrhemstad Does it make sense to you?

Reply:
Misunderstood your comments. Should be OK right now.